index.es5.js

```diff
@@ -0,0 +1,66 @@
+/*!
+ * vinyl-yaml-data | MIT (c) Shinnosuke Watanabe
+ * https://github.com/shinnn/vinyl-yaml-data
+*/
+'use strict';
+
+var path = require('path');
+var Transform = require('stream').Transform;
+
+var BufferStreams = require('bufferstreams');
+var objectPath = require('object-path');
+var replaceExt = require('replace-ext');
+var tryStreamPush = require('try-stream-push');
+var yaml = require('js-yaml');
+
+module.exports = function vinylYamlData(options) {
+  options = options || {};
+
+  return new Transform({
+    objectMode: true,
+    transform: function transform(file, enc, cb) {
```

Comment:
This line won't work on Node v0.x because the [simplified stream constructor API](https://nodejs.org/api/stream.html#stream_simplified_constructor_api) is only supported on Node v4+.